screens/login.js

```diff
@@ -39,13 +43,30 @@ const Login = () => {
                 "text-center text-lg font-bold mb-2 text-green-500"
               )}
             >
-              I forgot my password
+              Forgot your password?
             </Text>
           </TouchableOpacity>
-          <RoundButton style={tailwind("bg-green-500 mb-2")} title="login" />
+
+          <TouchableOpacity
+            onPress={() => {
+              fetch("http://localhost:8000/api/auth/login", {
+                method: "POST",
+                headers: { "Content-Type": "application/json" },
+                body: JSON.stringify({
+                  email: "email@domain.com",
```

Comment:
The email and password fields are hardcoded, they should be dynamic based on used input.